src/extension/editor.ts

```diff
@@ -267,7 +339,7 @@ export class TabbedZone extends Zone {
         return super.innerHeight;
     }
 
-    createWidget(styleClass: string = 'grid-preview') {
+    createWidget(styleClass: string = 'tile-preview grid-preview') {
```

Comment:
Question: is the class `tile-preview` declared by gnome-shell?

Reply:
Yeah, exactly: the class `tile-preview` is declared by gnome-shell. For reference, it is used by the class [TilePreview](https://github.com/GNOME/gnome-shell/blob/7c4b1d4ae62cfc6c5b0637819d465ce8968bd944/js/ui/windowManager.js#L388).

Since I want to add the border-radius property only to gSnap's zones and not to gnome's tiles, I give the class `grid-preview` too.

Reply:
Ok. I'd add a comment for future reference, so we know it is not declared in our extension!

Reply:
Seems like a great idea!